src/connectors/range-slider/connectRangeSlider.js

```diff
@@ -121,7 +121,7 @@ export default function connectRangeSlider(renderFn) {
         } else if (stats.min !== undefined && stats.min !== null) {
           min = stats.min;
         } else {
-          min = -Infinity;
+          min = 0;
```

Comment:
I don't get this change, when you want to select to complete range of results keep in mind they're might be lower than zeros values. Or am I missing something?

Reply:
My bad, it was fixed two commits later: https://github.com/algolia/instantsearch.js/pull/2408/commits/ca2ea036053aeed375db6579b84639b04df23aa6